backend/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Hosts the backend for LAKTA Mobile App",
   "main": "index.js",
   "scripts": {
-    "start": "node ./index.js",
-    "dev": "nodemon ./index.js",
+    "start": "node --experimental-specifier-resolution=node ./index.js",
```

Comment:
Added `--experimental-specifier-resolution=node` because to automatically add the .js extensions if needed while importing es6 modules